client/resource_manager_client.go

```diff
@@ -67,7 +68,11 @@ func (c *client) gRPCErrorHandler(err error) {
 // ListResourceGroups loads and returns all metadata of resource groups.
 func (c *client) ListResourceGroups(ctx context.Context) ([]*rmpb.ResourceGroup, error) {
 	req := &rmpb.ListResourceGroupsRequest{}
```

Comment:
We can create the request struct after `c.resourceManagerClient()`.

Reply:
done